### PR TITLE
Proof of `rdist_of_neg_le` 

### PR DESCRIPTION
**Pull request by pitmonticone**
- [x] Proof of lemma `rdist_of_neg_le` 
- [x] Added `ProbabilityTheory.IdentDistrib.inv` to port to Mathlib (see associated [PR](https://github.com/leanprover-community/mathlib4/pull/12918)). EDIT: Now merged.

Co-authored-by: Lorenzo Luccioli <lorenzoluccioli@gmail.com>

### CODE REVIEW

File: PFR/MoreRuzsaDist.lean [hunk 1]
```diff
@@ -82,10 +82,36 @@ variable {Ω Ω' Ω'' Ω''' G T : Type*}
 
 variable {X : Ω → G} {Y : Ω' → G} {Z : Ω'' → G} [FiniteRange X] [FiniteRange Y] [FiniteRange Z]
 
+-- Mathlib PR #12918 has been merged.
+@[to_additive]
+lemma ProbabilityTheory.IdentDistrib.inv {α : Type*} {β : Type*} {γ : Type*}
+    [MeasurableSpace α] [MeasurableSpace β] [MeasurableSpace γ] {μ : Measure α} {ν : Measure β}
+    {f : α → γ} {g : β → γ} [Inv γ] [MeasurableInv γ] (h : IdentDistrib f g μ ν) :
+    IdentDistrib f⁻¹ g⁻¹ μ ν := h.comp measurable_inv
```
Comment (YaelDillies) [inline PFR/MoreRuzsaDist.lean:90]: You should merge master into your branch to remove the lemma

Comment (pitmonticone) [reply PFR/MoreRuzsaDist.lean:90]: Ok @YaelDillies , we'll do it in a few minutes.

Comment (pitmonticone) [reply PFR/MoreRuzsaDist.lean:90]: I've also noticed that the checks for this draft PR are failing too.

Comment (YaelDillies) [reply PFR/MoreRuzsaDist.lean:90]: Uh that's because `says` is a pain. I'll fix it tomorrow

Comment (pitmonticone) [reply PFR/MoreRuzsaDist.lean:90]: Ok, thanks.

**Pull request merged**